cms_notification/models/res_users.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017-2018 Simone Orsi
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, fields, api
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    has_unread_notif = fields.Boolean(
+        string='Has unread notif',
+        compute='_compute_has_unread_notif',
+        readonly=True,
+    )
+
+    @api.multi
+    @api.depends()
```

Comment:
`depends` on what? :)

Reply:
on something that is not directly related, so you leave it empty.